package haptix.comm.msgs;

import "hxTransform.proto";

message hxCamera
{
  required hxTransform transform = 1;
}
